frappe/utils/data.py

```diff
@@ -258,6 +258,17 @@ def global_date_format(date, format="long"):
 	formatted_date = babel.dates.format_date(date, locale=(frappe.local.lang or "en").replace("-", "_"), format=format)
 	return formatted_date
 
+def is_weekday(date):
+	date = getdate(date)
+	return date.weekday() not in [5, 6]
+
+def next_weekday_date(date):
+	date = getdate(date)
+	date = add_days(date, 1)
+	while is_weekday(date) is False:
+		date = add_days(date, 1)
+	return formatdate(date)
```

Comment:
@netchampfaris hows this look? Thanks for your patience.

Reply:
I don't agree with formatdate here, just return the date object as-is, formatting should not be implicit!